tools/mkdeps.c

```diff
@@ -737,8 +737,8 @@ static void do_dependency(const char *file)
           *dotptr = '\0';
         }
 
-      snprintf(tmp, NAME_MAX + 6, " -MT %s%c%s%s ",
-               g_objpath, separator, objname, g_suffix);
+      snprintf(tmp, NAME_MAX + 6, " -MT %d-%s%c%s%s ",
+               file_idx, g_objpath, separator, objname, g_suffix);
```

Comment:
why need add a index here?

Reply:
This is not needed anymore. I removed the change.